profiles/SampleMoonMage-setup.yaml

```diff
@@ -5,6 +5,13 @@
 # 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
 # 2 Additional: Astrology, Scholarship
 
+# High-level example YAMLS:
+#  - Jonas
```

Comment:
Isn't `Jonas` also in the Necro high-level examples?

Reply:
Yeah, that's why he's added!

Reply:
Oh, wait, shit. I meant Crannach... too many alts! Thanks, good catch!